markitdownwordpress.php

```diff
@@ -33,7 +33,7 @@
     /*
      * Disable the wordpress visual editor, in Markdown writing we don't need it
      */
-    add_filter('user_can_richedit', create_function('', 'return false;'), 50);
+    add_filter('user_can_richedit', function() {return false;}, 50);
```

Comment:
I think you should use the core fuction `__return_false` so you can reduce this hook to
`add_filter('user_can_richedit', '__return_false', 50);`